py/examples/tour.py

```diff
@@ -39,7 +39,10 @@ async def start(self):
         if self.is_app:
             self.process = subprocess.Popen(
                 [sys.executable, '-m', 'uvicorn', '--port', _app_port, f'examples.{self.name}:main'],
-                env=dict(H2O_WAVE_EXTERNAL_ADDRESS=f'http://{_app_host}:{_app_port}', **env))
```

Comment:
suggested change without typo is https://github.com/h2oai/wave/compare/master...am/tour_encoding_issue